.github/workflows/build-images-ci.yaml

```
@@ -244,6 +244,30 @@ jobs:
             NOSTRIP=1
             OPERATOR_VARIANT=${{ matrix.name }}
 
+      - name: Install Bom and generate SBOM
+        shell: bash
+        run: | 
+          curl -L https://github.com/kubernetes-sigs/bom/releases/download/v0.3.0/bom-linux-amd64.tar.gz -o bom 
+          tar -xvf bom
```

[Nit 2/3] ...and then extract directly to `/usr/bin`...
```suggestion
          sudo tar xzvfC bom-linux-amd64.tar.gz /usr/local/bin
          rm bom-linux-amd64.tar.gz{,.sha256sum}
```

---

Corrected following previous discussion:
```suggestion
          sudo tar xzvfC bom-linux-amd64.tar.gz /usr/local/bin
          rm bom-linux-amd64.tar.gz
```